### PR TITLE
Updates code to reflect the new brand name Boon AI.

### PR DESCRIPTION
**Pull request by damusnet**
First pass at renaming. We need some clarity on the domain to complete.

Also fixes a missing `<title />` on the Login page.

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ![Django Backend](https://github.com/Zorroa/scrounger/workflows/Django%20Backend/badge.svg) ![React Frontend](https://github.com/Zorroa/scrounger/workflows/React%20Frontend/badge.svg)
 # What is Scrounger?
 
-Scrounger is an open source content search application built by [Zorroa](https://zorroa.com), the makers of a GUI-driven machine learning integration platform. The app is designed to let its users experience what it’s like to load their media assets and run an ML-powered  search within a custom app, in under an hour.
+Scrounger is an open source content search application built by [Boon AI](https://zorroa.com), the makers of a GUI-driven machine learning integration platform. The app is designed to let its users experience what it’s like to load their media assets and run an ML-powered  search within a custom app, in under an hour.
```
Comment (octaviaaris) [inline README.md:4]: What about a slight reword? It's inaccurate to say it's built by Boon AI bc of the whole dba discussion, no?

```suggestion
Scrounger is an open source content search application built by the makers of [Boon AI](https://zorroa.com), a GUI-driven machine learning integration platform. Scrounger is designed to let its users experience what it’s like to load their media assets and run an ML-powered  search within a custom app, in under an hour.
```

File: adobe_premiere_plugin/CSXS/manifest.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
         </Lifecycle>
         <UI>
           <Type>Panel</Type>
-          <Menu>Scrounger by Zorroa</Menu>
+          <Menu>Scrounger by Boon AI</Menu>
```
Comment (octaviaaris) [inline adobe_premiere_plugin/CSXS/manifest.xml:38]: ?

```suggestion
          <Menu>Scrounger powered by Boon AI</Menu>
```

File: README.md (hunk 1 above)
Comment (damusnet) [reply README.md:4]: I like this, but I feel like that's the part where @danny-zorroa wants to involve marketing/copywriting, so let's see what he says.

File: adobe_premiere_plugin/README.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ ln -s "$(pwd)" ~/Library/Application\ Support/Adobe/CEP/extensions/scrounger
 You may have to restart the `cfprefsd` process and/or reboot your computer.
 
 5. Open Adobe Premiere Pro and click the Window menu > Extensions > Scrounger by
```
Comment (octaviaaris) [inline adobe_premiere_plugin/README.md:64]: ?

```suggestion
5. Open Adobe Premiere Pro and click the Window menu > Extensions > Scrounger powered by
```

File: adobe_premiere_plugin/client/index.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
     <meta name="viewport" content="width=device-width" />
 
-    <title>Scrounger by Zorroa</title>
+    <title>Scrounger by Boon AI</title>
```
Comment (octaviaaris) [inline adobe_premiere_plugin/client/index.html:8]: ?

```suggestion
    <title>Scrounger powered by Boon AI</title>
```

File: adobe_premiere_plugin/client/index.html [hunk 5]
```diff
@@ -16,7 +16,7 @@
   <body class="text-white bg-gray-900 h-screen w-screen">
     <div class="py-4 px-2 xl:px-4">
       <h1 class="flex flex-col leading-none text-xl">
-        Scrounger<span class="text-sm text-green-500">by Zorroa</span>
+        Scrounger<span class="text-sm text-green-500">by Boon AI</span>
```
Comment (octaviaaris) [inline adobe_premiere_plugin/client/index.html:19]:
```suggestion
        Scrounger<span class="text-sm text-green-500">powered by Boon AI</span>
```

File: react_frontend/pages/_app.js [hunk 6]
```diff
@@ -13,23 +14,29 @@ class MyApp extends App {
...
-          }}
-        >
-          <div className="h-screen">
-            <div className="flex flex-col items-center w-full h-full">
-              <Header />
-
-              <Component {...pageProps} />
+      <>
+        <Head>
+          <title>Scrounger by Boon AI</title>
```
Comment (octaviaaris) [inline react_frontend/pages/_app.js:19]: ?

```suggestion
          <title>Scrounger powered by Boon AI</title>
```

File: react_frontend/src/Header/index.js [hunk 7]
```diff
@@ -11,7 +11,7 @@ const Header = () => {
       <div className="flex flex-row flex-wrap sm:flex-no-wrap justify-between py-4 w-full">
         <h1 className="flex flex-col leading-none text-xl pr-2">
           Scrounger
-          <span className="text-sm text-green-500">by Zorroa</span>
+          <span className="text-sm text-green-500">by Boon AI</span>
```
Comment (octaviaaris) [inline react_frontend/src/Header/index.js:14]: ?

```suggestion
          <span className="text-sm text-green-500">powered by Boon AI</span>
```

**Review by octaviaaris — APPROVED** [review]
suggested a few wording changes to align with the mental model that Boon AI is a product, not an entity.

File: README.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
-![Django Backend](https://github.com/Zorroa/scrounger/workflows/Django%20Backend/badge.svg) ![React Frontend](https://github.com/Zorroa/scrounger/workflows/React%20Frontend/badge.svg)
+![Django Backend](https://github.com/TheBoonAI/scrounger/workflows/Django%20Backend/badge.svg) ![React Frontend](https://github.com/TheBoonAI/scrounger/workflows/React%20Frontend/badge.svg)
```
Comment (octaviaaris) [inline README.md:1]: "The" Boon? I thought it was just "Boon AI"

File: README.md [hunk 9]
```diff
@@ -98,8 +98,8 @@ It is a Django/uWSGI application served by a gunicorn server on port 8000.
 To get started extending any of these components view the README.md found in each of their 
 respective folders for more information.
 
-[nginx_server/](https://github.com/Zorroa/scrounger/tree/main/nginx_server)
+[nginx_server/](https://github.com/TheBoonAI/scrounger/tree/main/nginx_server)
```
Comment (octaviaaris) [inline README.md:101]: Same question as above.

File: README.md [hunk 10]
```diff
@@ -98,8 +98,8 @@ It is a Django/uWSGI application served by a gunicorn server on port 8000.
...
 
-[react_frontend/](https://github.com/Zorroa/scrounger/tree/main/react_frontend)
+[react_frontend/](https://github.com/TheBoonAI/scrounger/tree/main/react_frontend)
 
-[django_backend/](https://github.com/Zorroa/scrounger/tree/main/django_backend)
+[django_backend/](https://github.com/TheBoonAI/scrounger/tree/main/django_backend)
```
Comment (octaviaaris) [inline README.md:105]: Same question as above.

File: adobe_premiere_plugin/README.md [hunk 11]
```diff
@@ -29,7 +29,7 @@ and other [CEP Resources](https://github.com/Adobe-CEP/CEP-Resources).
 1. Clone the project:
 
 ```
-git clone git@github.com:Zorroa/scrounger.git && cd scrounger/adobe_premiere_plugin
+git clone git@github.com:TheBoonAI/scrounger.git && cd scrounger/adobe_premiere_plugin
```
Comment (octaviaaris) [inline adobe_premiere_plugin/README.md:32]: Same question as above on "The" or no "The"

File: react_frontend/README.md [hunk 12]
```diff
@@ -73,7 +73,7 @@ lower mental overhead.
 ### Installation
 
 Clone the project:
-`git clone git@github.com:Zorroa/scrounger.git && cd scrounger/react_frontend`
+`git clone git@github.com:TheBoonAI/scrounger.git && cd scrounger/react_frontend`
```
Comment (octaviaaris) [inline react_frontend/README.md:76]: Same question as above.

File: README.md (hunk 8 above)
Comment (damusnet) [reply README.md:1]: I mean, look at the url on the current webpage where we are having this exchange 😛 

I assume "BoonAI" was not available when @danny-zorroa made the change (@danny-boonai?)

**Pull request merged**